File-Formats/PDF/Convert-HTML-To-PDF/net-core.md

```diff
@@ -49,17 +50,18 @@ Add a new action method ExportToPDF in HomeController.cs and include the below c
 
```

Comment:
Continue numbering for rest of the stpes

Reply:
added numbering for rest of the steps